### PR TITLE
Add udica templates

### PR DESCRIPTION
**Pull request by 0xC0ncord**
This is a reopen of #474 to add base container templates for [udica](https://github.com/containers/udica). Additional comments can be found in the previous PR.

### CODE REVIEW

**Review by pebenito — CHANGES_REQUESTED** [review]
Please add an install target in the `Makefile`.

**Comment by 0xC0ncord** [comment]
> Please add an install target in the `Makefile`.

Install target added as well as a test in the Github workflow.

File: Makefile [hunk 1]
```diff
@@ -655,6 +668,7 @@ bare: clean
 	$(verbose) rm -f $(mod_conf)
 	$(verbose) rm -f $(booleans)
 	$(verbose) rm -fR $(htmldir)
+	$(verbose) rm -fR $(udicatemplatesdir)
```
Comment (pebenito) [inline Makefile:671]: I don't think this should be added because `make bare` doesn't delete installed files.

Comment (0xC0ncord) [reply Makefile]: Good point. I've removed this.

**Pull request merged**